src/components/table/ProbeTableView.svelte

```diff
@@ -70,9 +71,7 @@
     data={selectedData}
     {aggregationLevel}
     colorMap={probeType === 'categorical' ? colorMap : percentileLineColorMap}
-    visibleBuckets={probeType === 'categorical'
-      ? visibleBuckets
-      : [5, 25, 50, 75, 95]}
+    visibleBuckets={probeType === 'categorical' ? visibleBuckets : PERCENTILES}
```

Comment:
This doesn't seem *quite* equivalent to your previous commit -- does non-categorical imply `METRICS_WITH_PERCENTILE_DATA`? https://github.com/mozilla/glam/pull/1649/commits/72ef1b9d8ab36241646f039599dafb545c7112d0